examples/redux/ngrx-actions/app/index.ts

```diff
@@ -1,11 +1,23 @@
-import { bootstrap } from '@angular/platform-browser-dynamic';
-import { provide } from '@angular/core';
-import { provideStore } from '@ngrx/store';
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { StoreModule } from '@ngrx/store';
 
 import { SimpleNgrx } from './containers/app-container';
+import { Counter } from './components/counter-component';
 import { counterReducer } from './reducers/counter-reducer';
 
-bootstrap(SimpleNgrx, [
-  provideStore({ counter: counterReducer })
-]);
-
+@NgModule({
```

Comment:
What we recommend now in the [readme for ng2-redux](https://github.com/angular-redux/ng2-redux/blob/master/README.md) is to configure the store in the module, instead of a component.

ie)

```js
@NgModule({
  /* ... */
  imports: [ /* ... */, NgReduxModule ]
})
export class AppModule {
  constructor(ngRedux: NgRedux<IAppState>) {
    ngRedux.configureStore(rootReducer, {}, [ createLogger() ]);
  }
}
```